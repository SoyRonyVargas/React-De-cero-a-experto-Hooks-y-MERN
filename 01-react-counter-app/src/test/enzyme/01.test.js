import { shallow } from 'enzyme'
import PrimeraAppApp from '../../PrimeraApp'

describe('Primer test', () => {
	test('should', () => {
		const saludo = 'Hola xdxd'

		const wrapper = shallow(<PrimeraAppApp saludo={saludo} />)

		expect(wrapper).toMatchSnapshot()
	})
	test('Debe de mostrar el subtitulo por props', () => {
		const saludo = 'esto es un titulo prop'
		const subtitulo = 'esto es un subtitulo prop'
		const wrapper = shallow(
			<PrimeraAppApp saludo={saludo} subtitulo={subtitulo} />
		)
		const textoParrafo = wrapper.find('p').text()

		console.log(textoParrafo)

		expect(textoParrafo).toBe(subtitulo)
		// expr
		// console.log(textoParrafo)
		// expect(textoParrafo)
	})
})
