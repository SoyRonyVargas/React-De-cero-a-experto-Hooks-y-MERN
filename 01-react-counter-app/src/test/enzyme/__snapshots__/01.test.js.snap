// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Primer test should 1`] = `
<Fragment>
  <h1>
     
    Hola xdxd
    !!! 
  </h1>
  <p>
    Soy un subtitulo
  </p>
</Fragment>
`;
